test/functional/cmdLineTests/build.xml

```diff
@@ -51,7 +51,7 @@
 					<fileset dir="." includes="*/build.xml" />
 				</subant>
 			</else>
-		</if>
+		</if>			
```

Comment:
There is no change for this file other than the extra space and the copyright year. Please revert.